test/runtime/runtime_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2017 XLAB, Ltd.
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+
+package runtime_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/mikelangelo-project/capstan/runtime"
+	. "gopkg.in/check.v1"
+)
+
+func Test(t *testing.T) { TestingT(t) }
+
+type testingRuntimeSuite struct{}
+
+var _ = Suite(&testingRuntimeSuite{})
+
+func (s *testingRuntimeSuite) TestPrependEnvsPrefix(c *C) {
+	m := []struct {
+		comment     string
+		cmd         string
+		env         []string
```

Comment:
Check [this comment](https://github.com/mikelangelo-project/capstan/pull/22/files#diff-4fe6c3a7f14439cfa03cf4bea9d12627R28)